app/assets/stylesheets/contribute.scss

```diff
@@ -65,6 +65,10 @@
   }
 }
 
+.justified-paragraph {
+  text-align: center;
+}
+
```

Comment:
the class name should be like `centered-paragraph`, as it has property `text-align: center`